tests/search/update_manager_test.py

```diff
@@ -0,0 +1,59 @@
+from tests.unit_test_base import UnitTestBase
+import tests.metadata_utils as utils
+from tests.search.test_wrapper import TestWrapper as SearchTestWrapper
+
+
+class UpdateManagerTest(UnitTestBase):
+    def setUp(self):
+        self.test_wrapper = SearchTestWrapper()
+        self.update_manager = self.test_wrapper.search_container.update_manager
```

Comment:
This serves a dual purpose.  Instead can you break out the functionality to instantiate a search_container with fake credentials?  It could go in a test base and then get injected into this test_wrapper.